mattermost.scm

```diff
@@ -604,6 +604,44 @@ a colon-separated pair of hours and minutes."
           (render-json
             `(("text" . ,(format #false "I couldn't understand the reminder: `~a`" text)))))))))
 
+(define (subcommand-update command-tokens) 
+  (match command-tokens
+    (("add" . handles)
+      (render-json
+        `(("text" . ,(string-join (add-handles handles)
+                                   ", ")))))
+    (("list")
+      (render-json
+        `(("text" . ,(list-handles)))))  
+    (("remove" . handles)
+      (render-json
+        `(("text" . ,(string-join (remove-handles handles)
+                                   ", ")))))
+    (("choose" . n-list)
+      (let ((choose-string (if (null? n-list)
```

Comment:
This is probably unsafe, there are no checks that this is actually a number. If it isn't the result would probably be an unhandled error.